arm_positions: [0.07, -0.72, -0.41, 2.27, 0.27, 1.33, -1.48]
head_positions: [0.0, -0.8]
gripper_positions: [0.05, 0.05]
torso_position: 0.16
